packages/frontend/src/ui/pagination/styles.module.css

```diff
@@ -0,0 +1,40 @@
+.root {
+    @apply flex gap-2;
+}
+
+.arrow {
+    @apply mx-2;
+}
+
+.arrowLeft {
+    @apply transform rotate-90;
+}
+
+.arrowRight {
+    @apply transform -rotate-90;
+}
+
+.item {
+    @apply w-9 h-9 rounded-md flex items-center justify-center transition-colors duration-200;
```

Comment:
`ease-in-out` too.

Reply:
Addressed in e3a2263d1d5914abd841a5695dcc92f0d2856e17